content/documents/sso-via-openid-connect.mdx

```diff
@@ -219,4 +220,18 @@ Now once login via OpenID is successful OrderCloud will redirect your user to th
 
 https://my-buyer-application.com?token={yourordercloudtoken}&refresh={yourordercloudrefreshtoken}
 
-> Note: You can only receive a refresh token if you have enabled your api client to allow refresh tokens by setting a refresh token duration.
\ No newline at end of file
+> Note: You can only receive a refresh token if you have enabled your api client to allow refresh tokens by setting a refresh token duration.
+
+## Custom Error Redirect
+
+If you wish to redirect a user to a custom page if an error occurs during authentication, you can add a value to `CustomErrorUrl` within your OpenID Connect Configuration.
+
+If an error occurs, OrderCloud will redirect the user to that custom error url.
+
+For example:
+
+```
+https://my-buyer-application.com/error?ErrorMessage={0}
+
+```
+The error message `ErrorMessage={0}` will be replaced with the error text from OrderCloud. 
```

Comment:
Did you mean to write that `{0}` will be replaced with the error text from OrderCloud?

Reply:
Yes, corrected it. 

@crhistianramirez - please let me know if you have any other suggestions.